src/iidm/NodeBreakerVoltageLevelViews.cpp

```diff
@@ -206,7 +210,7 @@ unsigned long NodeBreakerViewImpl::getNodeCount() const {
 }
 
 stdcxx::const_range<unsigned long> NodeBreakerViewImpl::getNodes() const {
-    return m_voltageLevel.getGraph().getVertices();
+    return m_voltageLevel.getNodes();
```

Comment:
I don't understand this change: the implementation will differ from the java code

Reply:
Ok reverted.